code/game/mecha/mecha_control_console.dm

```diff
@@ -21,8 +21,8 @@
 			var/answer = TR.get_mecha_info()
 			if(answer)
 				dat += {"<hr>[answer]<br/>
-						  <a href='?src=[REF(src)];send_message=[REF(TR)]'>Send message</a><br/>
-						  <a href='?src=[REF(src)];get_log=[REF(TR)]'>Show exosuit log</a> | <a style='color: #f00;' href='?src=[REF(src)];shock=[REF(TR)]'>(EMP pulse)</a><br>"}

```

Comment:
Why remove the log function?

Reply:
no clue why tg did it, added it back

Reply:
Honestly I don't remember removing it, so it was probably an oversight on my part and no one said anything because no one else ever used it or even knew that feature existed.